main.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+from data_collection.create_map import setting_region_of_interest, get_city_coordinates
+from src.constants import Default_City, Default_Year
+
+
+if __name__ == "__main__":
```

Comment:
probably not needed anymore as this module will always be directly run, see also: https://docs.python.org/3/library/__main__.html#id1